.github/workflows/compatibility_tests.yml

```diff
@@ -41,10 +41,10 @@ jobs:
       run: |
         # 1. only install the dev requirements on top of what is in the cellxgene pip package
         sudo apt-get update && sudo apt-get install -y libhdf5-serial-dev
-        sed -i 's/-r requirements.txt//' server/requirements-dev.txt
-        pip install -r server/requirements-dev.txt
+        sed -i 's/-r requirements.txt//' backend/czi_hosted/requirements-dev.txt
```

Comment:
I think the compatibility tests should probably be testing against the desktop version, not the czi-hosted backend. This is because we pin specific versions for the czi-hosted backend but the compatibility tests are meant to be for folks using the CLI who may have varying versions of anndata installed.

Reply:
doh! updated